libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5128,14 +5100,19 @@ MAV_RESULT GCS_MAVLINK::handle_command_storage_format(const mavlink_command_int_
 MAV_RESULT GCS_MAVLINK::handle_command_int_packet(const mavlink_command_int_t &packet)
 {
     switch (packet.command) {
+#if HAL_MOUNT_ENABLED
     case MAV_CMD_DO_SET_ROI:
     case MAV_CMD_DO_SET_ROI_LOCATION:
         return handle_command_do_set_roi(packet);
-#if HAL_MOUNT_ENABLED
     case MAV_CMD_DO_SET_ROI_SYSID:
-        return handle_command_do_set_roi_sysid(packet);
     case MAV_CMD_DO_SET_ROI_NONE:
-        return handle_command_do_set_roi_none();
+        {
+            AP_Mount *mount = AP_Mount::get_singleton();
```

Comment:
All of this should be moved into `void GCS_MAVLINK::handle_command_int(const mavlink_message_t &msg)` instead